pkgs/development/mobile/androidenv/androidsdk.nix

```diff
@@ -246,6 +246,11 @@ stdenv.mkDerivation rec {
             ln -sf $i $out/bin/$(basename $i)
         fi
     done
+
+    mkdir -p $out/libexec/licenses
```

Comment:
Everywhere: https://github.com/search?q=8933bad161af4178b1185d1a37fbf41ea5269c55&ref=simplesearch&type=Code&utf8=%E2%9C%93 from TravisCI scripts up to Kotlin source code.